src/pyBiodatafuse/constants.py

```diff
@@ -438,3 +438,127 @@
+    "anatomical_entity_node": "http://semanticscience.org/resource/SIO_001262",
+    "tested_substance_node": "http://semanticscience.org/resource/SIO_010038",
+    "source_database": f"{NAMESPACE_BINDINGS['sio']}SIO_000750",
+    "experimental_process_node": "http://www.ebi.ac.uk/efo/EFO_0002694",
+    "pathway_node": f"{NAMESPACE_BINDINGS['obo']}PW_0000001",
+    "adverse_event_node": f"{NAMESPACE_BINDINGS['obo']}OAE_0000001",
+    "ensemble": "http://identifiers.org/ensembl/",
+    "ncbi_disease": "https://www.ncbi.nlm.nih.gov/medgen/",
+    "article": f"{NAMESPACE_BINDINGS['obo']}IAO_0000013",
+    "protein_node": "http://purl.obolibrary.org/obo/NCIT_C17021",
+    "ppi_node": "http://purl.obolibrary.org/obo/NCIT_C18469",
+    "drug_node": "http://semanticscience.org/resource/SIO_010038",
+    "approved_drug": "http://purl.obolibrary.org/obo/NCIT_C172573",
+    "aid": "http://purl.obolibrary.org/obo/CLO_0037244",
+    "developmental_stage_node": "http://purl.obolibrary.org/obo/NCIT_C43531",
+}
+
+# PREDICATES
+PREDICATES = {
+    "sio_refers_to": f"{NAMESPACE_BINDINGS['sio']}SIO_000628",
+    "sio_has_measurement_value": f"{NAMESPACE_BINDINGS['sio']}SIO_000216",
+    "sio_has_source": f"{NAMESPACE_BINDINGS['sio']}SIO_000253",
+    "sio_is_associated_with": f"{NAMESPACE_BINDINGS['sio']}SIO_001403",
+    "sio_has_value": f"{NAMESPACE_BINDINGS['sio']}SIO_000300",
+    "sio_has_input": f"{NAMESPACE_BINDINGS['sio']}SIO_000230",
+    "sio_has_output": f"{NAMESPACE_BINDINGS['sio']}SIO_000229",
+    "chebi_inchi": "http://purl.obolibrary.org/obo/chebi/inchi",
+    "chebi_smiles": "http://purl.obolibrary.org/obo/chebi/smiles",
+    "cheminf_compound_id": "http://semanticscience.org/resource/CHEMINF_000140",
+    "sio_is_part_of": f"{NAMESPACE_BINDINGS['sio']}SIO_000068",
+    "has_gene_count": "",  # TODO gene count
+    "precedes": "http://semanticscience.org/resource/SIO_000248",
+    "is preceded by": "http://semanticscience.org/resource/SIO_000248",
+    "sio_has_part": f"{NAMESPACE_BINDINGS['sio']}SIO_000028",
+    "negatively_regulates": f"{NAMESPACE_BINDINGS['obo']}RO_0002449",
+    "phase": "http://purl.obolibrary.org/obo/PATO_0000083",
+    "has_gene_template": "http://purl.obolibrary.org/obo/pr#has_gene_template",
+}
+
+# Classes for clinical phases
+
+CLINICAL_PHASES = {
+    "1.0": "http://purl.obolibrary.org/obo/OPMI_0000368",
+    "2.0": "http://purl.obolibrary.org/obo/OPMI_0000369",
+    "3.0": "http://purl.obolibrary.org/obo/OPMI_0000370",
+    "4.0": "http://purl.obolibrary.org/obo/OPMI_0000371",
+}
+
+# GO Types
+GO_TYPES = {
+    "C": "http://purl.obolibrary.org/obo/GO_0005575",
+    "P": "http://purl.obolibrary.org/obo/GO_0008150",
+    "F": "http://purl.obolibrary.org/obo/GO_0003674",
+}
+
+# Compound MoA
+MOAS = {
+    "activates": "http://purl.obolibrary.org/obo/RO_0018027",  # agonist
+    "inhibits": "http://purl.obolibrary.org/obo/RO_0018029",  # antagonist
+}
+
+# Data sources
+DATA_SOURCES = {
```

Comment:
call database names from the variable list, to make sure if they change latter, it would be reflected here too